.github/dependabot.yml

```diff
@@ -23,6 +24,7 @@ updates:
     schedule:
       interval: "daily"
     labels:
+      - '6.16.z'
```

Comment:
How about removing 6.12.z label from this matrix, as we're not going to cherrypick to 6.12.z?

Reply:
Thanks removed